classes/DepositPoint.inc.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * @file plugins/generic/sword/classes/DepositPoint.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class DepositPoint
+ * @ingroup plugins_generic_sword_classes
+ *
+ * @brief Deposit point entity.
+ */
+
+class DepositPoint extends DataObject {
+	/**
+	 * Get the ID of the deposit point.
+	 * @return int
+	 */
+	public function getId() {
+		return $this->getData('depositPointId');
+	}
+
+	/**
+	 * Set the ID of the deposit point
+	 * @param $feedId int
+	 */
+	public function setId($depositPointId) {
```

Comment:
(Going to the default implementation should help make this double-call redundant.)